api/Dockerfile

```diff
@@ -4,20 +4,14 @@ WORKDIR /app
 
 RUN apt-get update && apt-get install -y gcc build-essential libpq-dev musl-dev python3-dev
 
-# Set up venv
-RUN python -m venv /opt/venv
-# Make sure we use the virtualenv:
-ENV PATH="/opt/venv/bin:$PATH"
-
-COPY requirements.txt .
-
-# Make sure we are running latest pip and setuptools to avoid potential security warnings
 RUN pip install --upgrade pip
 RUN pip install --upgrade setuptools
+RUN pip install poetry
```

Comment:
Please update `api/Makefile` to acommodate for new dependency management.

I'd also prefer to have Poetry version pinned there. To me, the most sensible way to do this would be to set a `POETRY_VERSION` env var, as it transparently gets picked up by the [official Poetry installer](https://python-poetry.org/docs/#installing-with-the-official-installer) — which IMO should be used locally and likely in the Docker build too. The reason for this is the fact that the official installer places Poetry in its isolated venv automatically, which eliminates the need for us to maintain a venv for the application (see comment below).

Reply:
Resolved the comments.